table-project/src/utils/fetchUsers.ts

```diff
@@ -0,0 +1,10 @@
+type User = {
+    id: string
+    name: string
+    website: string
+}
+
+export default async function fetchUsers() {
+    const res = await fetch('https://jsonplaceholder.typicode.com/users');
+    return res.json() as Promise<User[]>;
+}
```

Comment:
Define a single file that holds the function for fetching data + propagate a type to get nice suggestions for autocomplete.